evm_arithmetization/src/cpu/kernel/asm/transactions/type_0.asm

```diff
@@ -18,131 +18,97 @@ global process_type_0_txn:
     %stack (rlp_addr, len) -> (rlp_addr)
 
     // stack: rlp_addr, retdest
+    // Store the nonce position in the RLP segment.
```

Comment:
Hmm maybe my review should have been written differently, but this seems to imply we're writing the nonce position in memory. I think the use of `store` may be confusing? Or maybe I'm just getting a bit tired.

Reply:
(if you're fine with it, feel free to merge)

Reply:
I agree, it's confusing. I'll use `Keep track of the X position` for more clarity.